helm_deploy/hmpps-manage-users-api/values.yaml

```diff
@@ -46,6 +46,9 @@ generic-service:
     DELIUS_ROLES_MAPPINGS_MRDBT003: "ROLE_MAKE_RECALL_DECISION_PPCS"
     DELIUS_ROLES_MAPPINGS_MRDBT004: "ROLE_MAKE_RECALL_DECISION_ADMIN"
     DELIUS_ROLES_MAPPINGS_MRDBT005: "ROLE_MAKE_RECALL_DECISION_SUPPORT"
+    MARD_DUTY_MANAGER_MRDBT006: "ROLE_MARD_DUTY_MANAGER"
```

Comment:
Can you pls follow the patter? 
DELIUS_ROLES_MAPPINGS_*
DELIUS_ROLES_MAPPINGS_MRDBT006

Reply:
Done